meiki_server/notes/notes_test.go

```diff
@@ -75,15 +75,25 @@ var note3 = notes.Note{
 	Content:  "What is my purpose?; You are a dummy for this test; Oh my god",
 }
 
+var invalidNote = notes.Note{
```

Comment:
Why is this variable out in the global scope (I'm missing something)

Reply:
should have been `const`, these are fixtures

Reply:
Only primitive types can be a global const (https://stackoverflow.com/questions/38542588/making-constant-global-variables-of-a-custom-struct-type), so seems like we ate stuck with var here :(